float32 ee_x
float32 ee_y
float32 ee_z
float32 ee_phi
---
float32 joint_1
float32 joint_2
float32 joint_3
float32 joint_4
float32 left_gripper_joint
